content/docs/actors/pay_per_result_actors.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Actors paid per result
+description: Learn how actors that are paid per result work. Find out how this differ from the traditional "pay for usage" model and how this benefits you.
+
+menuWeight: 7.4
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actors/pay-per-result-actors
+---
+
+This page explains a new pricing model for actors in the [Apify store](https://apify.com/store) that allows you to pay only for the results an actor generates and not the underlying platform usage. This makes it transparent and easy to estimate upfront costs. If you have any feedback or would like to ask something, please join our [Discord](https://discord.gg/qkMS6pU4cF) community and let us know!
+
+# Traditional approach - paying for usage
+
+Traditionally, public actors in the [Apify Store](https://apify.com/store) come in two flavors: free and [paid actors](https://docs.apify.com/platform/actors/paid-actors). The latter requires a monthly fee to rent the actor and use it after its trial period. When you run either a free or paid actor from the store, they generate platform usage that is subtracted from your prepaid usage, which is either $5 on the free plan or prepaid usage given by your subscription plan.
+
+![Free and paid actors in Apify Store]({{@asset actors/images/free_vs_paid_actors.webp}})
+
+This platform usage includes components such as [compute](https://docs.apify.com/platform/actors/running/compute-units), operations on [storages](https://docs.apify.com/platform/storage), and usage of [residential proxies](https://docs.apify.com/platform/proxy/residential-proxy) or [SERPs](<https://docs.apify.com/platform/proxy/google-serp-proxy>). You can see the usage your actor run generated on the run detail page.
+
+![Example run usage]({{@asset actors/images/example_run_usage.webp}})
+
+In this model, it is quite difficult to know beforehand exactly how much usage the actor will generate when you run it so you cannot easily estimate your costs upfront.
+
+## Paying only for results instead
+
+With **pay per result** model you will only pay for the results that an actor returns when you run it, and you will not be charged for the underlying platform usage. Technically, **results** you are charged for are the clean items in the default [dataset](https://docs.apify.com/platform/storage/dataset) created by the actor run.
+
+<!-- TODO - add a bit about the ability to set the maximum limits actor should return -->
+
+### How do I know an actor is paid per result?
+Actor that is paid per result will display the price per 1,000 items in the store.
```

Comment:
An actor that is paid per result will display the price per 1,000 items in the store.